src/views/AccountMyFormsDref/ActiveDrefTable/index.tsx

```diff
@@ -229,6 +229,7 @@ function ActiveDrefTable(props: Props) {
                     const canCreateFinalReport = !has_final_report
                         && (applicationType === 'DREF' || applicationType === 'OPS_UPDATE')
                         && (is_published ?? false)
+                        && (item.type_of_dref_display !== 'Loan')
```

Comment:
can't we use `item.type_of_dref` instead of `item.type_of_dref_display`?  
Comparision using enum would be better.